app/views/requests/comments/_list.html.erb

```diff
@@ -1,10 +1,10 @@
 
 <ul id="request_annotations_<%= @request.id %>" class="list-group">
-  <% if @request.comments.size > 0 %>
+  <% if @comments.present? %>
     <% @comments.each do |comment| %>
       <li class="list-group-item">
         <%= render partial: "shared/comment" ,
-                   locals: { comment: comment, commentable: @request.becomes(Request), update_target: "request_annotations_#{@request.id}" }%>
+                   locals: { comment: comment, update_target: "request_annotations_#{@request.id}" }%>
```

Comment:
We're no longer just looking at request comments, so don't override the commentable. I've moves the 'becomes' bit into the partial itself.